_layouts/indicator.html

```diff
@@ -171,5 +173,29 @@ <h4>{{ t.indicator.sub_categories }}</h4>
   </div>
 </div>
 
-{% include footer.html %}
+<style type="text/css">
+  #chartControls[show="1"]{display: block;}
+  #chartControls[show="0"]{display: none;}
+</style>
 
+<script>
```

Comment:
Could this be added in the [indicator view file](https://github.com/open-sdg/open-sdg/blob/master/_includes/assets/js/indicatorView.js)?